src/app/containers/suggestion-of-activities/suggestion-of-activities.component.html

```diff
@@ -4,28 +4,24 @@
   </div>
   <alg-error
     *ngIf="state.isError"
+    class="light"
+    icon="ph-duotone ph-warning-circle"
+    buttonStyleClass="alg-button primary refresh-button light-border"
     i18n-message message="Error while loading the activities"
     [showRefreshButton]="true"
     refreshButtonType="refresh"
-    (refresh)="refresh()"
+    (refresh)="refresh($event)"
   ></alg-error>
-  <div *ngIf="state.isReady && state.data.length === 0" i18n>
+  <div class="empty-list" *ngIf="state.isReady && state.data.length === 0" i18n>
     There are no activities
```

Comment:
"There are no activities linked to this group." (or "user" if it applies to user as well)